databases/week2/School database.sql

```diff
@@ -0,0 +1,32 @@
+CREATE DATABASE school_database;
+USE school_database;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE `class` (
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+      `name` VARCHAR(40) NOT NULL,
+      `begins` DATETIME NOT NULL ,
```

Comment:
This could be just a `DATE` type since you probably don't need to know the time when it starts. But it's also good as is :)